internal/bminventory/inventory.go

```diff
@@ -1694,7 +1694,7 @@ func (b *bareMetalInventory) generateClusterInstallConfig(ctx context.Context, c
 		installerReleaseImageOverride = *defaultArchImage.URL
 	}
 
-	if err := b.generator.GenerateInstallConfig(ctx, cluster, cfg, *releaseImage.URL, installerReleaseImageOverride); err != nil {
+	if err := b.generator.GenerateInstallConfig(ctx, cluster, cfg, *releaseImage.URL, installerReleaseImageOverride, serviceBaseURL, b.authHandler.AuthType()); err != nil {
```

Comment:
base url and auth type both are set once on startup, maybe it should be part of the configuration to the generator?

Reply:
done